Examples.md

```diff
@@ -319,7 +321,7 @@ spec:
           restartPolicy: Never
           containers:
             - name: run-backup-cron
-              image: clickhouse/clickhouse-client:latest
+              image: clickhouse/clickhouse-server:latest
```

Comment:
let's add comment here about clickhouse-client doesn't support anymore, to avoid comments about "ah why you run second clickhouse-server?"

actually old clickhouse-clients should connect to latest clickhouse-server, protocol compatible, which kind of issue did you look here?

Reply:
ok. got it
https://hub.docker.com/r/clickhouse/clickhouse-client/tags
doesn't support arm64, lets add comment here